src/components/Tooltip/__stories__/tooltip.stories.mdx

```diff
@@ -0,0 +1,205 @@
+import Tooltip from "../Tooltip";
+import TooltipReference from "./TooltipReference";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import { Hide, Subitems, Bolt } from "../../Icon/Icons"
+import Icon from "../../Icon/Icon.jsx"
+import Button from "../../Button/Button.jsx";
+import { modifiers } from "./helper.js";
+import { Link } from "../../../storybook/components";
+import { CHIP, LABEL, TIPSEEN } from "../../../storybook/components/related-components/component-description-map";
+import "./tooltip.stories.scss";
+
+<Meta
+  title="Components/Tooltip"
+  component={ Tooltip }
+/>
+
+<!--- Component template -->
+
+export const tooltipTemplate = (args) => {
+  return (
+    <div className="monday-storybook-tooltip_overview">
+      <Tooltip
+        shouldShowOnMount={args.shouldShowOnMount}
+        content={args.content}
+        modifiers={args.modifiers}
```

Comment:
please add note above the modifiers prop (same note I wrote in the other comments)

Reply:
also, you can already set the modifiers like this: modifiers={modifiers}
and to take their value from the args.